src/LibYear/Output/ConsoleOutput.cs

```diff
@@ -0,0 +1,73 @@
+﻿using LibYear.Core;
+using Spectre.Console;
+
+namespace LibYear.Output;
+
+internal sealed class ConsoleOutput : IOutput
```

Comment:
All new classes should be `public` so they can be tested

Reply:
I'm happy to do this because it's your codebase, but isn't this also why there's the [InternalsVisibleToAttribute](https://learn.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.internalsvisibletoattribute?view=net-8.0) so that you can test internal concerns without exposing them to other people?

Reply:
Yeah, not a fan of that technique myself, but to each their own!